src/panels/config/cloud/account/cloud-webhooks.ts

```diff
@@ -195,9 +195,17 @@ export class CloudWebhooks extends LitElement {
   }
 
   private async _fetchData() {
-    this._localHooks = isComponentLoaded(this.hass!, "webhook")
-      ? await fetchWebhooks(this.hass!)
-      : [];
+    if (!isComponentLoaded(this.hass!, "webhook")) {
+      this._localHooks = [];
```

Comment:
```suggestion
      this._localHooks = [];
      return
```
We don't need to go further here if the integration is not loaded.